.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature request
+    url: https://curl.haxx.se/mail/
+    about: To propose new features or enhancements, please bring that discussion to a suitable curl mailing list.
+  - name: Security vulnerability
+    url: https://hackerone.com/curl
```

Comment:
Would it make sense to link to https://curl.haxx.se/dev/secprocess.html instead?

Reply:
I don't think so. The secprocess page is more about describing how we approach it, if someone wants to report an issue I figured the hackerone page is more direct.